js/piwik.js

```diff
@@ -2706,8 +2706,19 @@ if (typeof window.Piwik !== 'object') {
         }
 
         function isInsideAnIframe () {
-            if (isDefined(windowAlias.frameElement)) {
-                return (windowAlias.frameElement && String(windowAlias.frameElement.nodeName).toLowerCase() === 'iframe');
+            var frameElement;
+
+            try {
+                // If the parent window has another origin, then accessing frameElement
+                // throws an Error in IE. Compare issue #10105.
+                frameElement = windowAlias.frameElement;
+            } catch (e) {
```

Comment:
The jslint problem is same variable is used twice.  if you rename `e` to `error` it should fix jslint.